stdlib/3/dbm/dumb.pyi

```diff
@@ -0,0 +1,31 @@
+
+import sys
+from typing import Union, MutableMapping, Iterator, Optional, Type
+from types import TracebackType
+
+if sys.version_info >= (3, 8):
+    from typing import Final
+else:
+    from typing_extensions import Final
+
+_KeyType = Union[str, bytes]
+_ValueType = Union[str, bytes]
+
+error: Final[Type[OSError]]
```

Comment:
Just `error = OSError` works and creates a type alias, which is preferable because mypy will let you use it in a type annotation.